app/thirdparty/java/other/writeily/widget/WrMarkorWidgetProvider.java

```diff
@@ -29,6 +29,11 @@
 
 public class WrMarkorWidgetProvider extends AppWidgetProvider {
     public static final String WIDGET_PATH = "WIDGET_PATH";
+    private int requestCode = 0;
+
+    private int nextCode() {
```

Comment:
how about random code instead? (Random.nextint()). Should be more likely to happen 0 twice (due value forgotten by rotation) than hitting random twice